.github/workflows/composer-validate.yml

```diff
@@ -0,0 +1,30 @@
+name: "Check validity of composer.json"
```

Comment:
@alcaeus: I assume this was done for the PIE package, but did you plan to copy this workflow to [PHPLIB](https://github.com/mongodb/mongo-php-library/tree/master/.github/workflows) as well?

Reply:
Yes, this was done to make sure we're alerted to changes to the pie schema, especially as it may change during the development process. I can copy the workflow over to PHPLIB 👍